packages/broker/src/plugins/storage/StorageConfig.ts

```diff
@@ -100,13 +85,13 @@ export class StorageConfig {
         this.poller = setTimeout(() => this.poll(pollInterval), pollInterval)
     }
 
-    hasStream(stream: StreamPart): boolean {
-        const key = getKeyFromStream(stream.id, stream.partition)
-        return this.streamKeys.has(key)
+    hasSPID(spid: Protocol.SPID): boolean {
```

Comment:
Pondering whether full name would be clearer?
```suggestion
    hasStreamPartition(spid: Protocol.SPID): boolean {
```